src/godirect.js

```diff
@@ -35,20 +36,40 @@ const godirect = {
   * This invokes the navigator.bluetooth.requestDevice method and returns the selected device as a Device instance.
   * This can only be invoked via a user interaction (e.g. within a click event) otherwise you'll get a security warning.
   * @name selectDevice
+  * @param {bool} bluetooth
   * @returns {Promise} Promise object represents a Device instance
   */
-  async selectDevice() {
-    if (!navigator.bluetooth) {
-      return Promise.reject(new Error('No Web Bluetooth support.'));
-    }
+  async selectDevice(bluetooth = true) {
```

Comment:
This means we're limiting it to only bluetooth or usb devices. I think the selection UI could allow us to display both types, right?